File: src/main/resources/templates/DemoServiceImpl.java
package {{dubboProviderPackageName}};

import java.util.Date;

import {{dubboPackageName}}.{{serviceSimpleName}};

import com.alibaba.dubbo.config.annotation.Service;

import org.springframework.stereotype.Component;

@Service(version = "{{dubboServiceVersion}}")
public class {{serviceSimpleName}}Impl implements {{serviceSimpleName}} {

	@Override
    public String sayHello(String name) {
        return "Hello, " + name + ", " + new Date();
    }

}

File: src/main/resources/templates/DemoServiceImpl.kt
package {{dubboPackageName}};

import java.util.Date;

import com.alibaba.dubbo.config.annotation.Service;

@Service(version = "{{dubboServiceVersion}}")
class {{serviceSimpleName}}Impl : {{serviceSimpleName}} {

    override fun sayHello(name: String): String {
        return "Hello, " + name + ", " + Date()
    }

}